kodilla-collections/src/main/java/com/kodilla/collections/interfaces/homework/Audi.java

```diff
@@ -0,0 +1,25 @@
+package com.kodilla.collections.interfaces.homework;
+
+public class Audi implements Car{
+    int speed;
+
+    public Audi(int speed) {
+        this.speed = speed;
+    }
+
+    @Override
+    public void increaseSpeed() {
+        speed +=100;
+        return;
+    }
+
+    @Override
+    public void decreaseSpeed() {
+        speed -=10;
```

Comment:
czy predkosc moze byc ujemna? Gdybym te metode wywolal np. 100 razy, to mialnbym ujemna predkosc